app/renderer/src/assets/stylesheets/common.global.less

```diff
@@ -99,3 +99,8 @@ body::-webkit-scrollbar-corner {
 .ant-tabs-tab {
   font-size: 14px;
 }
+
+.search-word-highlighted {
```

Comment:
I would suggest moving this back to the Inspector-specific stylesheet, since this will not be used for the Session page

Reply:
I moved it to common module to improve the testability and since it is a common class which can be used across any module in future it makes sense to keep it in common css.

Reply:
See my other comment - if nothing is exported from the JSX, this is not needed for test purposes. Also, this file does not define any custom styles, but only overrides default ones. All custom styles are contained in the page-specific stylesheets.

Reply:
Got it. i was banging my head around looking for ways to enable the testability. This make more sense and will make the necessary changes.